app/models/container_service.rb

```diff
@@ -1,5 +1,6 @@
 class ContainerService < ApplicationRecord
   include CustomAttributeMixin
+  include SupportsFeatureMixin
```

Comment:
`ContainerService` include `CiMixin` that uses `SupportsFeatureMixin`.

The `included do` of `CiMixin` will fail on `supports :capture` unless the `supports` means something.